examples/routing/compare_results.jl

```diff
@@ -2,49 +2,52 @@ using CSV
 using DataFrames
 using Statistics
 
-# Define paths
-basin_path = "/groups/esm/achiang/ClimaRivers.jl/data/routing/simulations/simulations_lv05/simulation_gamma-IRF"
-channel_path = "/groups/esm/achiang/ClimaRivers.jl/data/routing/simulations/simulations_lv05/gamma_IRF"
-hillslope_path = "/groups/esm/achiang/ClimaRivers.jl/data/routing/simulations/simulations_lv05/gamma_IRF"
-output_path = "/groups/esm/achiang/ClimaRivers.jl/examples/routing/csv_files"  # Directory to save output CSVs
+# build environment
+data_file_path = joinpath(@__DIR__, "..", "..", "data", "routing")
+
+# build static environment
+@info "reading data files from $(data_file_path)"
+maurico_path = joinpath(
```

Comment:
Could we maybe leave out Mauricio's name? Maybe we can call it a "reference" simulation?